x-pack/plugins/observability_onboarding/e2e/cypress/e2e/home.cy.ts

```diff
@@ -10,6 +10,27 @@ describe('[Observability onboarding] Landing page', () => {
     cy.loginAsElastic();
   });
 
+  describe('Entry point', () => {
+    it('from setup guides', () => {
```

Comment:
nit: as a rule of thumb the `describe` should explain the "act" eg `when clicking on the logs card`, and the `it` should explain the assertion `navigates to the observability onboarding page`.

It is also fine to combine the two:

```suggestion
    it('when clicking on the logs card the user is navigated to the observability onboarding page', () => {
```

Reply:
Addressed in [f8f7f5d](https://github.com/elastic/kibana/pull/164561/commits/f8f7f5d8f5139414e8a0ff4d9d046cbd9f456252).